packages/lighthouse-service/src/property-manager/resolver.ts

```diff
@@ -54,6 +54,14 @@ export const PropertyResolver = {
       const savedProperty = await mongoosePropertyDoc.save();
       return populateMongooseDocWithUser(savedProperty);
     },
+    async createLHProject(root: any, args: any, ctx: any) {
```

Comment:
I think this needs to be in audit manager. 
But let it be in property manager it will avoid merge conflict with my current changes of LH backend. Ill move it to audit manager and property manager will get removed.

Reply:
I have created a new PR with the Lighthouse service changes and kept these as part of the property manager in LH service.